tools/tcptop.py

```diff
@@ -88,9 +88,9 @@ def range_check(string):
 BPF_HASH(ipv4_recv_bytes, struct ipv4_key_t);
 
 struct ipv6_key_t {
-    u32 pid;
     unsigned __int128 saddr;
     unsigned __int128 daddr;
+    u32 pid;
     u16 lport;
     u16 dport;
 };
```

Comment:
Great. Thanks for the fix. In order to make it fully correct in case other variables on the stack, could you change to
```
struct ipv6_key_t {
    unsigned __int128 saddr;
    unsigned __int128 daddr;
    u32 pid;
    u16 lport;
    u16 dport;
    u64 __pad__;
};
```

Reply:
Sure, no problem.